### ISSUE TITLE
Delete community health files from individual repos

### ISSUE THREAD

**Issue by reefdog** [tvkitchen/meta#4]
_This issue is blocked until we have the org-level `.github` repo._

After #3, we should delete community health files from individual repos that don't require differences from the shared versions.

**📣 Discussion Topic: 📣** The above is pretty clear for magic GitHub files like issue and PR templates, but non-magical files like CONTRIBUTING and CODE_OF_CONDUCT can also be inherited. This is nice since they inevitably are full of identical boilerplate that are a pain to keep up-to-date across repos, but also means the files won't appear in the individual repo file browsers or be included in repo downloads/clones. [The introduction blog post says](https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/):

> While the file itself won’t appear in the file browser or Git history for each repository, it will be surfaced throughout developers’ workflows, such as when opening a new issue or when viewing the Community Profile, just as if it were committed to the repository directly.

Is it sufficient that those sort of things show up only in the [not entirely discoverable Community section](https://github.com/tvkitchen/tv-kitchen/community)?

**Comment by slifty** [comment]
![image](https://user-images.githubusercontent.com/208884/87213336-37aa4000-c2f2-11ea-8787-67bebb484e1f.png)
